src/chromo/models/pythia6.py

```diff
@@ -13,6 +13,9 @@ def _charge_init(self, npart):
         # TODO accelerate by implementing this loop in Fortran
         return np.fromiter((self._lib.pychge(ki) / 3 for ki in k), np.double)
 
+    def _add_init_beam_info(self):
+        pass
+
```

Comment:
Remove this and the other empty implementations to declutter the code after remove the `abstractmethod` decorator in the base class. We only want to make a method abstract if it is absolutely required for every derived class to implement it. If a default implementation exists and can be used for a subset of all derived classes, then we only override it for those where it is necessary and use the (empty) default implementation for the others.

Reply:
Removed.